worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -421,9 +421,9 @@ namespace RTC
 		{
 			// Allocate a new storage item.
 			storageItem = StorageItemPool.Allocate();
-			// Memory is not initialized in any way, initialize with default values it to make sure
-			// contents is correct.
-			*storageItem = StorageItem{};
+
+			// Memory is not initialized in any way, reset it.
+			std::memset(storageItem, 0, sizeof(StorageItem));
```

Comment:
I'm not sure this is a good idea. All zeroes is not necessarily a valid invariant of the data structure. It can be, but I would prefer to use default values and not manipulating memory directly.

Reply:
That's correct. I't's now properly handled.